modules/misunderstood/src/views/full/index.tsx

```diff
@@ -0,0 +1,236 @@
+import { AxiosStatic } from 'axios'
+import { Container, SidePanel, SplashScreen } from 'botpress/ui'
+import React from 'react'
+
+import { FLAGGED_MESSAGE_STATUS, FlaggedEvent, ResolutionData } from '../../types'
+
+import style from './style.scss'
+import ApiClient from './ApiClient'
+import MainScreen from './MainScreen'
+import SidePanelContent from './SidePanel'
+
+const getDefaultLanguage = (languages: string[]) => {
+  if (languages.includes('en')) {
+    return 'en'
+  }
+  return languages[0]
+}
+
+const INITIAL_STATUS = FLAGGED_MESSAGE_STATUS.new
+
+interface MainViewProps {
+  bp: {
+    axios: AxiosStatic
+  }
+}
+
+interface MainViewState {
+  loaded: boolean | null
+  languages: string[] | null
+  language: string | null
+  eventCounts: { [status: string]: number } | null
+  selectedStatus: FLAGGED_MESSAGE_STATUS
+  events: FlaggedEvent[] | null
+  selectedEventIndex: number | null
+  selectedEvent: FlaggedEvent | null
+}
+
```

Comment:
Small comment, since we try to keep components in individual files, we are using `interface Props` and `interface State` to keep thing simple, instead of naming them by the class they are being used for